README.md

```diff
@@ -225,6 +225,7 @@ Build automation tools
 * [Shake](https://github.com/ndmitchell/shake) - Define robust build systems akin to GNU Make using Haskell.
 * [Make](https://www.gnu.org/software/make/) - The GNU Make build system.
 * [Prodmodel](https://github.com/prodmodel/prodmodel) - Build system for data science pipelines.
+* [Just](https://github.com/casey/just) - Command and recipe runner similar to Make, built in Rust
```

Comment:
It would nicer to list it alphabetically

Reply:
ok done

Reply:
Thanks